tests/phpunit/tests/rest-api/rest-attachments-controller.php

```diff
@@ -566,6 +568,60 @@ public function test_get_items_valid_date() {
 		$this->assertEquals( $id2, $data[0]['id'] );
 	}
 
+	/**
+	 * @ticket 50617
+	 */
+	public function test_get_items_invalid_modified_date() {
+		$request = new WP_REST_Request( 'GET', '/wp/v2/media' );
+		$request->set_param( 'modified_after', rand_str() );
+		$request->set_param( 'modified_before', rand_str() );
+		$response = rest_get_server()->dispatch( $request );
+		$this->assertErrorResponse( 'rest_invalid_param', $response, 400 );
+	}
+
+	/**
+	 * @ticket 50617
+	 */
+	public function test_get_items_valid_modified_date() {
+		$id1 = $this->factory->attachment->create_object(
+			$this->test_file,
+			0,
+			array(
+				'post_date'      => '2016-01-01T00:00:00Z',
+				'post_mime_type' => 'image/jpeg',
+				'post_excerpt'   => 'A sample caption',
+			)
+		);
+		$id2 = $this->factory->attachment->create_object(
+			$this->test_file,
+			0,
+			array(
+				'post_date'      => '2016-01-02T00:00:00Z',
+				'post_mime_type' => 'image/jpeg',
+				'post_excerpt'   => 'A sample caption',
+			)
+		);
+		$id3 = $this->factory->attachment->create_object(
+			$this->test_file,
+			0,
+			array(
+				'post_date'      => '2016-01-03T00:00:00Z',
+				'post_mime_type' => 'image/jpeg',
+				'post_excerpt'   => 'A sample caption',
+			)
+		);
+		$this->update_post_modified( $id1, '2016-01-15T00:00:00Z' );
```

Comment:
```suggestion
		$this->update_post_modified( $id1, '2016-01-15 00:00:00' );
```

This function accepts dates as `YYYY-MM-DD HH:MM:SS`.

Reply:
I changed all of my dates to follow the correct format.

Reply:
Looks good!